### PR TITLE
Add how to manually update the Translation Status Overview Issue

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- New or updated content

#### Description

This PR adds instructions on how to manually update the Translation Status Overview Issue.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ For PRs that are translations to existing Docs content, including new page addit
 - If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`.
 - If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place. While we always prefer to have a review from a native speaker of the language, having translated docs with some errors is usually better than having no docs at all.
 
+#### Manual Translation Status Overview Issue Updates
+
+If a PR that should have been marked as a minor change was merged, the Translation Status Overview Issue will consider the page's translations as "needs updating". In case this occurs and there aren't any other updates needed for that translation, take the following steps to manually update the tracker:
```
Comment (sarah11918) [inline MAINTAINERS.md:98]:
```suggestion
If the Translation Status Overview Issue incorrectly shows "needs updating" for a page (e.g. a typo fix to an English page triggered the status update), take the following steps to manually update the tracker:
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -93,6 +93,16 @@ For PRs that are translations to existing Docs content, including new page addit
...
+
+- Open the [Translation Status Overview](https://github.com/withastro/docs/issues/438) Issue.
+- Click the "Edit" button.
+- Find the big JSON object at the end (i.e, the translation's internal progress data).
+- Replace the `lastMajorChange` value of the affected languages with today's date and hour.
```
Comment (sarah11918) [inline MAINTAINERS.md:103]: A code sample or screenshot might be nice here, but that can always be added later if we find people need more guidance following these instructions!

File: MAINTAINERS.md [hunk 3]
```diff
@@ -93,6 +93,16 @@ For PRs that are translations to existing Docs content, including new page addit
 - If you speak the language natively, check the content for accuracy. Note: some languages have created their own glossaries and/or language guides located in their language folder within `/src/i18n/`.
 - If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place. While we always prefer to have a review from a native speaker of the language, having translated docs with some errors is usually better than having no docs at all.
 
+#### Manual Translation Status Overview Issue Updates
```
Comment (sarah11918) [inline MAINTAINERS.md:96]:
```suggestion
#### Correcting an Incorrect Translation Status
```

File: MAINTAINERS.md [hunk 4]
```diff
@@ -93,6 +93,16 @@ For PRs that are translations to existing Docs content, including new page addit
...
+
+If a PR that should have been marked as a minor change was merged, the Translation Status Overview Issue will consider the page's translations as "needs updating". In case this occurs and there aren't any other updates needed for that translation, take the following steps to manually update the tracker:
+
+- Open the [Translation Status Overview](https://github.com/withastro/docs/issues/438) Issue.
+- Click the "Edit" button.
+- Find the big JSON object at the end (i.e, the translation's internal progress data).
+- Replace the `lastMajorChange` value of the affected languages with today's date and hour.
+
+The next time a PR gets merged, the translation tracking script will be rerun and update the overview issue, removing the page from the "needs updating" section.
```
Comment (sarah11918) [inline MAINTAINERS.md:105]:
```suggestion
This process tells the tracker that the page was updated "now." The next time a PR gets merged, the translation tracking script will be rerun and the page will appear to have been updated, removing its "needs updating" icon.
```

File: MAINTAINERS.md [hunk 5]
```diff
@@ -93,6 +93,16 @@ For PRs that are translations to existing Docs content, including new page addit
...
 - If you do not speak the language natively, and the PR has not had any recent activity, you can use online translation tools (e.g. Google Translate) and scan the results for anything that looks wildly out of place. Also, visit the page in the PR’s Netlify deploy preview to verify that nothing is visually out of place. While we always prefer to have a review from a native speaker of the language, having translated docs with some errors is usually better than having no docs at all.
 
+#### Manual Translation Status Overview Issue Updates
+
+If a PR that should have been marked as a minor change was merged, the Translation Status Overview Issue will consider the page's translations as "needs updating". In case this occurs and there aren't any other updates needed for that translation, take the following steps to manually update the tracker:
+
+- Open the [Translation Status Overview](https://github.com/withastro/docs/issues/438) Issue.
+- Click the "Edit" button.
+- Find the big JSON object at the end (i.e, the translation's internal progress data).
```
Comment (sarah11918) [inline MAINTAINERS.md:102]:
```suggestion
- Find the big JSON object at the end (i.e. the translation's internal progress data).
```

**Review by sarah11918 — COMMENTED** [review]
Love this, @Yan-Thomas! Thanks so much for these helpful instructions!

As usual, I made some suggestions to make language very direct and sometimes even remove some of the "why" and technical "how" to make the commands stand out more.

But make no mistake, you make it SO EASY to do this, having it already written and thought out! This is so helpful!

File: MAINTAINERS.md (hunk 2 above)
Comment (Yan-Thomas) [reply MAINTAINERS.md:103]: I added a code sample with a very simplified view of the JSON object, let me know what you think!

**Comment by sarah11918** [comment]
LGTM!

**Pull request merged**